server/errorHandler.ts

```diff
@@ -17,7 +17,3 @@ export function createErrorHandler(production: boolean) {
     return res.render('pages/error')
   }
 }
-
-export default {
```

Comment:
we needed this strange `export default` stuff here when `app.js` was javascript, now it has been converted can be changed back into just one `export`